cibyl/utils/files.py

```diff
@@ -55,3 +55,15 @@ def get_first_available_file(filenames, file_check=is_file_available):
 
     # None of the files exist
     return None
+
+
+def get_file_name_from_path(path):
```

Comment:
Please add tests for windows like paths, like: "C:\Users\...\file.txt".

Reply:
I've added a couple of tests with windows-like paths